apps/judicial-system/web/src/components/InfoCard/InfoCardClosedIndictment.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useContext } from 'react'
```

Comment:
@Arnarkari93, what do you think about this, having separate InfoCard components based on the data in the InfoCard and then deciding which one to use on the Overview page?

Reply:
I think it is a good idea 👍